spec/meal_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+require_relative '../lib/meal.rb'
+
+describe Meal do
+
+  let (:duck_dish){ double(:dish, see: "44 : Duck a la banana : £4.50 : 2",
+    total: 9.00) }
+  let (:deep_dish){ double(:dish, see: "10 : Deep fried Mars bar : £0.10 : 10",
```

Comment:
Style/Blocks: Avoid using {...} for multi-line blocks.<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.